src/travel-aid/TravelAidScreenComponent.tsx

```diff
@@ -114,7 +121,21 @@ const TravelAidSection = ({
     fromQuayId,
   );
 
-  useTravelAidAnnouncements({status, focusedEstimatedCall});
+  let notices: NoticeFragment[] = [];
+  if (serviceJourney !== undefined) {
+    notices = getNoticesForServiceJourney(serviceJourney, fromQuayId) ?? [];
+  }
```

Comment:
Can this be simplified to this?

```suggestion
  const notices = getNoticesForServiceJourney(serviceJourney, fromQuayId);
```